packages/components/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 -   `FormTokenField`: Fix focus lost on tab when `__experimentalExpandOnFocus` is set ([#70591](https://github.com/WordPress/gutenberg/pull/70591)).
 -   `SelectControl`: Fix font-size for medium screens to ensure consistency with other inputs ([#70619](https://github.com/WordPress/gutenberg/pull/70619)).
 
+### Internal
+
+-   `SelectControl`: Move classnames to the root ([#70643](https://github.com/WordPress/gutenberg/pull/70643)).
```

Comment:
This is still a potentially breaking change for consumers, so I think we should give it more prominence than "Internal". How about a bug fix? I think we can classify this as a bug fix.